tests/mdev_configuration_allocation_test.go

```diff
@@ -56,18 +57,45 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Eventually(ThisPod(testPod), 3*time.Minute, 5*time.Second).Should(BeInPhase(k8sv1.PodSucceeded))
 	}
-	checkAllMDEVRemoved := func() {
-		check := fmt.Sprintf(`set -x
+
+	var latestPod *k8sv1.Pod
```

Comment:
nit: I would like to see all vars together at the beginnig :)

Reply:
I have intentionally kept it to a place where it is used in the hope nobody else will use it. I am not proud of the code and will take any suggestions on how to improve it.

Reply:
The only thing it comes in my mind is that you can pass `latestPod` to the `waitForPod` function. In this way the variable is not accessible from the entire test. But I am not sure of which one is better